processor/coralogixprocessor/config.go

```diff
@@ -0,0 +1,36 @@
+package coralogixprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/coralogixprocessor"
+
+import "fmt"
+
+type CacheConfig struct {
+	maxCacheSizeBytes int64 `mapstructure:"max_cache_size_bytes"`
+	maxCachedEntries  int64 `mapstructure:"max_cached_entries"`
+}
+
+type DatabaseBlueprintsConfig struct {
+	WithSampling bool `mapstructure:"with_sampling"`
+	CacheConfig  `mapstructure:"cache_config"`
+}
+
+type Config struct {
+	DatabaseBlueprintsConfig `mapstructure:"database_blueprints_config"`
```

Comment:
Do all of these need to be public? In general we want to have as few public members as possible.

Reply:
[70e11a5](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33458/commits/70e11a571a3a04acfeb8ee53d885f725c78d8275)